doc/plugins.md

```diff
@@ -139,6 +139,8 @@ See [Creating plugins][create] below.
     — highlight code blocks for **remark-react** with [lowlight](https://github.com/wooorm/lowlight)
 *   [`remark-redact`](https://github.com/seafoam6/remark-redact)
     — conceal text matching a regex
+*   [`remark-redactable`](https://github.com/code-dot-org/remark-redactable)
+    — write plugins to redact content from a markdown document, then restore it later
```

Comment:
```suggestion
    — write plugins to redact content from a markdown document,
    then restore it later
```

Oh no, now it complains about line length, how about this?

Reply:
D'oh!